include/bluetooth/buf.h

```diff
@@ -57,8 +57,7 @@ enum bt_buf_type {
  *
  *  @param type    Type of buffer. Only BT_BUF_EVT and BT_BUF_ACL_IN are
  *                 allowed.
- *  @param timeout Timeout in milliseconds, or one of the special values
- *                 K_NO_WAIT and K_FOREVER.
+ *  @param timeout Timeout in milliseconds, or 0, or SYS_FOREVER_MS.
```

Comment:
This is conflicting with the change here:
https://github.com/zephyrproject-rtos/zephyr/pull/24126/files#diff-3b88339f7f6c1a678793dbd25736263a

I suggest to revert the changes in this file.

Reply:
done